src/parser/monk/mistweaver/modules/features/StatValuesSpellInfo.js

```diff
@@ -134,4 +134,108 @@ export default {
     mastery: false,
     vers: true,
   },
+  [SPELLS.ANCIENT_TEACHINGS_OF_THE_MONASTERY_HEAL.id]: {
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: false,
+    vers: true,
+  },
+  [SPELLS.GUST_OF_MISTS_CHIJI.id]: { // Chi-ji's gusts
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: true,
+    vers: true,
+  },
+  [SPELLS.SOOTHING_BREATH.id]: { //Yu'lon's healing
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: true,
+    mastery: false,
+    vers: true,
+  },
+  [SPELLS.ENVELOPING_BREATH.id]: { 
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: true,
+    mastery: false,
+    vers: true,
+  },
+  [SPELLS.EXPEL_HARM.id]: { 
+    int: true,
+    crit: true,
+    hasteHpm: false,
+    hasteHpct: false,
+    mastery: false,
+    vers: true,
+  },
+  [SPELLS.TEAR_OF_MORNING_HEAL.id]: { 
+    int: true,
+    crit: true,
+    hasteHpm: true,
+    hasteHpct: true,
```

Comment:
hasteHpct and hasteHpm should be set to false here, both values are already being calculated by the proccing spells. No need to double dip.